app/packages/core/src/components/Modal/TooltipInfo.tsx

```diff
@@ -157,20 +162,20 @@ const useTarget = (field, target) => {
   return getTarget(field, target);
 };
 
-const AttrInfo = ({ label, children = null }) => {
+const AttrInfo = ({ label, infoType, children = null }) => {
   let entries = Object.entries(label).filter(
     ([k, v]) => "tags" !== k && !k.startsWith("_")
   );
   if (!entries || !entries.length) {
     return null;
   }
-
+  const defaultLabels = ["label", "confidence"]
   const defaults = entries.filter(([name]) =>
-    ["label", "confidence"].includes(name)
+    defaultLabels.includes(name)
   );
-
+  
   const other = entries.filter(
-    ([name]) => !["label", "confidence"].includes(name)
+    ([name]) => !([...defaultLabels, ...HIDDENLABELS[infoType], "attributes"].includes(name))
```

Comment:
Ah I forgot to mention the `attributes` parameter in my first review. Only the `Detection`, `Keypoint` and `Polyline` classes have a builtin `attributes` parameter, so this logic could just be added to `HIDDEN_LABELS` below

Reply:
Ah I see 👍